bootstrap.sh

```diff
@@ -28,6 +28,9 @@ echo -e "👉 Create symlink for pre-commit hooks"
 # Symlink needs to be ../../hooks and not ./hooks because git evaluates them in .git/hooks
 ln -sf ../../hooks/pre-commit.sh .git/hooks/pre-commit
 chmod +x .git/hooks/pre-commit
+ln -sf ../../hooks/post-rewrite.sh .git/hooks/post-rewrite
+chmod +x .git/hooks/post-rewrite
+
```

Comment:
I don't know enough about this but since it's `post-rewrite`, are we sure it work the same way we expect? aka - when the file is reformatter or when `Package.swift` is changed, the changes are staged and added to the current commit?

Reply:
Yes, exactly how you say. I tried this with changing Package.swift (deleting some excluded files) and then amending that into previous commit, works flawlessly. :)